elixir/.formatter.exs

```diff
@@ -1,5 +1,5 @@
 # Used by "mix format"
 [
-  inputs: ["{mix,.formatter}.exs", "{config,lib,test}/**/*.{ex,exs}"],
+  inputs: ["{mix,.formatter}.exs", "{config,test}/**/*.{ex,exs}", "lib/*.ex"],
```

Comment:
@fabriziosestito I think we should've put `lib/protobuf/*` here. Otherwise the `contract.ex` file is not formatted, and this file is not autogenerated.

Reply:
Ok, forget about it. I guess it is just the opposite. That it only format the contract.ex file hehe